public/style/base.css

```diff
@@ -0,0 +1,270 @@
+  margin: .5em;
+  padding: .5em;
+  margin-left: 15em;
+  color: white;
+}
+.bio {
+  width: 40em;
+  height: 20em;
+}
+
+main {
+  width: 77.75em;
+}
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: whitesmoke;
+  height: 8em;
+  width: 73em;
+  border-radius: .5em;
+  border: .5em double rgba(255,255,255,0.3);
+}
+.navItem {
+  font-size: 4em;
+}
+a {
+  text-decoration: none;
+  margin: 2em;
+  color: steelblue;
+}
+body {
+  margin: 0em;
+  min-height: 100vh;
+  width: 74em;
+}
+#meetTheTeam {
+  font-size: 9em;
+}
+h1 {
+  margin-right: .5em;
+  font-size: 2.25em;
+  margin-top: 0;
+  text-align: center;
+  color: white;
+  margin-top: .5em;
+}
+h2 {
+  color: red;
+  text-align: center;
+}
+
+input[type=checkbox] {
+  -ms-transform: scale(5); /* IE */
+  -moz-transform: scale(5); /* FF */
+  -webkit-transform: scale(5); /* Safari and Chrome */
+  -o-transform: scale(5); /* Opera */
+  padding: 1em;
+}
+
+.title {
+  font-size: 4em;
+  color: white;
+  margin-right: 1em;
+}
+.welcome {
+  background-color: lavender;
+  margin-left: 20%;
+  margin-right: 20%;
+}
+html {
+  margin: 0;
+  background-size: cover;
+  background: url("../images/kerryPark.jpg") no-repeat center center fixed;
+  background-blend-mode: hard-light;;
+  transition: 3s;
+}
+.intro {
+  color: white;
+  margin-right: 4em;
+  line-height: 2.5;
+}
+ul li {
+  display: inline-block;
+}
+#shruti, #ron, #sam {
+  width: 40em;
+  height: 40em;
+}
+#warning  {
+  display: none;
+  z-index: 1;
+  border-width: medium;
+  border-color: red;
+  background-color: white;
+  text-align: center;
+}
+#warning h2 {
+  z-index: 1;
+  padding: 1em;
+  color: red;
+}
+#warning p {
+  z-index: 1;
+  padding: 1em;
+  font-size: 1.15em;
+  border-width: medium;
+  border-color: red;
+}
+#placesTemplate{
+  display: inline-block;
+  overflow-x: scroll;
+  overflow-y: scroll;
+  float:right;
+  height: 40em;
+  width:35%;
+  background-color: lightblue;
+}
+#warning  {
+ display: none;
+ z-index: 1;
+ border-width: medium;
+ border-color: red;t
+}
+#warning h2 {
+ z-index: 1;
+ padding: 1em;
+ color: red;
+}
+#warning p {
+ z-index: 1;
+ padding: 1em;
+ font-size: 1.15em;
+ border-width: medium;
+ border-color: red;
+}
+div#texttop {
+  color: beige;
+  margin-top: 1%;
+  padding: 1rem;
+  font-family: sans-serif;
+  width: 78em;
+  height: 10em;
+}
+#initForm {
+  color: white;
+  width: 13em;
+  font-size: 6em;
+}
+#homeDiv {
+  width: 80em;
+  height: 100%;
+}
+#about {
+  width: 100%;
+  height: 100%;
+}
+#submitButt {
+  font-size: 1em;
+  padding: .1em;
+  margin-top: 1em;
+  margin-left: 4.4em;
+  text-align: center;
+  border-radius: .25em;
+  background-color: #B4BBCC;
+  box-shadow: 0em .05em .05em .05em #1B1B25;
+}
+#placesTemplate,#Toggle{
+   float:right;
+   width:33%;
+   display:inline-block;
+   height: 100%;
+ }
+#map-data-wrapper{
+  display:inline-block;
+}
+#Toggle{
+  float:right;
+}
+#primary_nav_wrap {
```

Comment:
There are a lot of ID's here in your CSS, and you can probably simplify this heavily by consolidating common CSS rules into class/other selectors and applying those classes to the necessary elements.